joplin/base/models.py

```diff
@@ -482,6 +481,52 @@ class DepartmentPageDirector(Orderable):
         FieldPanel('about'),
     ]
 
+class OfficialDocumentPage(JanisPage):
+    janis_url_page_type = "official_document"
+    base_form_class = custom_forms.OfficialDocumentPageForm
+
+    description = models.TextField(blank=True)
+
+    department = models.ForeignKey(
```

Comment:
I could be misreading this but I think we need to decide whether to use the department model (one to many) or allow multiple related departments. 

Since you made a related departments clusterable model, I'd assume we want a document page to potentially belong to multiple departments(which sounds like an organizational mess but shrug emoji), so we might want to stick to one option or the other? 

Any thoughts @briaguya ?

Reply:
multiple departments everywhere. I know we've done that for info/service pages, I figured we're keeping that pattern for documents pages too.

Reply:
OK, so we'll want to nix department being declared as a ForeignKey here then, yah?